ocs_ci/ocs/constants.py

```diff
@@ -818,7 +819,7 @@
 
 # PDB NAMES
 MDS_PDB = "rook-ceph-mds-ocs-storagecluster-cephfilesystem"
-OSD_PDB = "rook-ceph-osd-"
```

Comment:
this constant is being used by validate_pdb_creation and this is called during deployment. This can break the existing code, please check